components/org.wso2.carbon.identity.api.idle.account.identification/org.wso2.carbon.identity.api.idle.account.identification.v1/src/main/java/org/wso2/carbon/identity/api/idle/account/identification/v1/core/InactiveUsersManagementApiService.java

```diff
@@ -263,4 +307,80 @@ private void validateDatesCombination(LocalDateTime inactiveAfterDate, LocalDate
                     String.format(error.getDescription()));
         }
     }
+
+    /**
+     * Get the filter node as a list.
+     *
+     * @param filter value of the filter.
+     * @return node tree.
+     * @throws IdleAccountIdentificationClientException Error when validate filters.
+     */
+    private List<ExpressionNode> getExpressionNodes(String filter) throws IdleAccountIdentificationClientException {
+
+        // Filter example : isDisabled eq true.
+        List<ExpressionNode> expressionNodes = new ArrayList<>();
+        FilterTreeBuilder filterTreeBuilder;
+        try {
+            if (StringUtils.isNotBlank(filter)) {
```

Comment:
Do this check at first and return the empty list. Not required to be have in the try block.